src/Particle/ParticleSet.cpp

```diff
@@ -670,8 +670,11 @@ void ParticleSet::acceptMove_impl(Index_t iat, bool forward_mode)
   if (iat == activePtcl)
   {
     //Update position + distance-table
-    for (int i = 0, n = DistTables.size(); i < n; i++)
-      DistTables[i]->update(iat, forward_mode);
+    for (int i = 0; i < DistTables.size(); i++)
```

Comment:
General question on our loops over particle set updates: are they currently required to be sequential starting from i=0? if not, can we write the loops more generally?

Reply:
Order should not matter here and range based loop is better. It is probably good rainy day task reviewing all the loops in this file.